packages/mdc-drawer/component.ts

```diff
@@ -146,6 +146,8 @@ export class MDCDrawer extends MDCComponent<MDCDismissibleDrawerFoundation> {
       notifyOpen: () => this.emit(strings.OPEN_EVENT, {}, true /* shouldBubble */),
       trapFocus: () => this.focusTrap_!.activate(),
       releaseFocus: () => this.focusTrap_!.deactivate(),
+      addBodyClass: (className) => document.body.classList.add(className),
```

Comment:
We may have use cases where the inner elements of body are scrollable instead of the body itself.

We may be able to solve this problem by preventDefault'ing `touchmove` event on scrim element.

WDYT?

Reply:
I doubt that preventDefault on touchmove prevents overscrolling behavior since it's a css thing that browser relies on it. but maybe I'm wrong. so please give me tomorrow to test it too.

Reply:
sorry for delay I was busy with our project. I tried it. putting touchmove on scrim doesn't work but it works for mdc-dialog itself because the scrim is a sibling to main dialog content but mdc-dialog is the parent so the overscroll could be prevented from mdc-dialog root. if you prefer this way I'm going to apply changes. we should do this in many places I think, in select menu etc.

Reply:
@abhiomkar I applied the changes in a new commit please review and let me know if I should rebase and squash these 2 commits.